onnxruntime/test/ep_weight_sharing_ctx_gen/README.md

```diff
@@ -2,32 +2,31 @@
 
 This tool provides the way to generate Onnx models that wraps QNN context binary warpt with weight sharing enabled. The options to use with the tool are listed below:
 
-`ep_weight_sharing_ctx_gen [options...] model_1_path,model_2_path`
+```
+ep_weight_sharing_ctx_gen [options...] model1_path,model2_path
 
-./ep_weight_sharing_ctx_gen -e qnn -v -i "soc_model|60 htp_graph_finalization_optimization_mode|3" /mnt/c/model1.onnx,/mnt/c/model2.onnx
+Example: ./ep_weight_sharing_ctx_gen -i "soc_model|60 htp_graph_finalization_optimization_mode|3" -C "ep.context_node_name_prefix|_part1" ./model1.onnx,./model2.onnx
```

Comment:
still need to keep -e qnn, we are trying to enable this tool with other EPs

Reply:
ok. I copied it from the usage text directly. will update the usage text too.